doc/quick_tutorial.rst

```diff
@@ -17,62 +17,108 @@
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
-.. _quick_tutorial_ref:
+.. Edited October 1, 2020 - [USA]TechDude
 
 Quick Tutorial
-==============
-
-Using OpenShot is very easy, and this tutorial will take you through the basics in
-**under 5 minutes**. After this tutorial, you will be able to make a simple photo
+===============
+OpenShot is easy to use, and it has many advanced features.  This introductory 
```

Comment:
Should easy to use be easy-to-use everywhere mentioned? Also, I think I prefer the original text to the new one here. The new one seems harder to read... ?